www/src/views/community/community-header.js

```diff
@@ -0,0 +1,270 @@
+              "&:hover": {
+                backgroundColor: `initial`,
+              },
+            },
+          }}
+        >
+          Creators
+        </Link>
+        <nav
+          role="navigation"
+          css={{
+            display: `flex`,
+            justifyContent: `space-around`,
+            alignItems: `center`,
+          }}
+        >
+          <CommunityHeaderLink linkTo="/community/">All</CommunityHeaderLink>
+          <CommunityHeaderLink linkTo="/community/people/">
+            People
+          </CommunityHeaderLink>
+          <CommunityHeaderLink linkTo="/community/agencies/">
+            Agencies
+          </CommunityHeaderLink>
+          <CommunityHeaderLink linkTo="/community/companies/">
+            Companies
+          </CommunityHeaderLink>
+        </nav>
+        <div
+          className="community--filters"
+          css={{
+            display: `flex`,
+          }}
+        >
+          <label
+            className="label"
+            css={[styles.filter, forHire && styles.activeFilter]}
+          >
+            <input
+              type="checkbox"
+              name="forHire"
+              css={{
+                ...styles.input,
+              }}
+              checked={forHire}
+              onChange={() => this.props.applyFilter(`for_hire`)}
+              disabled={hiring}
+            />
+            For Hire
+          </label>
+          <label
+            className="label"
+            css={[styles.filter, hiring && styles.activeFilter]}
+          >
+            <input
+              type="checkbox"
+              name="hiring"
+              css={{
+                ...styles.input,
+              }}
+              checked={hiring}
+              onChange={() => this.props.applyFilter(`hiring`)}
+              disabled={forHire}
+            />
+            Hiring
+          </label>
+        </div>
+        <div
+          css={{
+            marginLeft: `auto`,
+            display: `flex`,
+            alignItems: `center`,
+          }}
+        >
+          <button
+            css={{
+              background: `transparent`,
+              border: `1px solid ${colors.lilac}`,
+              color: colors.lilac,
+              padding: `${rhythm(1 / 4)} 1rem`,
+              display: `flex`,
+              alignItems: `center`,
+              borderRadius: `2px`,
+              cursor: `pointer`,
+              marginRight: `${rhythm(1 / 4)}`,
+              "&:hover": {
+                backgroundColor: colors.gatsby,
+                color: `white`,
+              },
+            }}
+          >
+            <span
+              css={{
+                marginRight: `${rhythm(1 / 4)}`,
+              }}
+            >
+              Become a Creator
+            </span>
+            <img src={Arrow} alt="" css={{ marginBottom: 0 }} />
+          </button>
+          {/* // Search function is not implemented yet, do we want/need it here? */}
```

Comment:
Search seems likely to be complex enough for its own component. Or are you asking something different?

Reply:
Ah this was more of a question of, do we need search functionality on this page? Which we'll probably resolve in today's meeting